src/m365/spo/commands/site/site-commsite-enable.ts

```diff
@@ -64,10 +79,14 @@ class SpoSiteCommSiteEnableCommand extends SpoCommand {
   }
 
   public async commandAction(logger: Logger, args: CommandArgs): Promise<void> {
-    const designPackageId: string = args.options.designPackageId || '96c933ac-3698-44c7-9f4a-5fd17d71af9e';
+    const designPackageId = args.options.designPackageId || {
+      Topic: '96c933ac-3698-44c7-9f4a-5fd17d71af9e',
+      Showcase: '6142d2a0-63a5-4ba0-aede-d9fefca2c767',
+      Blank: 'f6cc5403-0d63-442e-96c0-285923709ffc'
+    }[args.options.designPackage || 'Topic'];
 
     if (this.verbose) {
-      logger.logToStderr(`Enabling communication site at ${args.options.url}...`);
+      logger.logToStderr(`Enabling communication site with design package '${args.options.designPackage || 'Topic'}' at '${args.options.url}'...`);
```

Comment:
This would not log options.designPackageId in the current state. Let's just log the local `designPackageId` variable.